libnetdata/socket/socket.c

```diff
@@ -369,6 +372,38 @@ static inline int bind_to_this(LISTEN_SOCKETS *sockets, const char *definition,
         port = e + 1;
         *e = '\0';
     }
+    e++;
+    while (*e!='\0' && *e!='=') e++;
+    if(*e == '=') {
+        *e='\0';
+        e++;
+        portconfig = e;
+        while (*e != '\0') {
+            if (*e == '|') {
+                *e = '\0';
+                acl_flags = acl_flags | read_acl(portconfig);
+                e++;
+                portconfig = e;
+                continue;
+            }
+            e++;
+        }
+        acl_flags = acl_flags | read_acl(portconfig);
+    } else {
+        acl_flags = WEB_CLIENT_ACL_DASHBOARD | WEB_CLIENT_ACL_REGISTRY | WEB_CLIENT_ACL_BADGE | WEB_CLIENT_ACL_MGMT | WEB_CLIENT_ACL_NETDATACONF | WEB_CLIENT_ACL_STREAMING;
```

Comment:
IMPORTANT:

So, the default ACL includes management?
How this will affect existing installations that will update their netdata?

I suggest that the default ACL will not include management.